src/routes/session/[id]/+page.svelte

```
@@ -9,300 +9,392 @@
+    if (!ticket || disposed) return;
+
+    const protocol = window.location.protocol === 'https:' ? 'wss:' : 'ws:';
+    const ws = new WebSocket(`${protocol}//${window.location.host}${wsPath}?ticket=${ticket}`);
+
+    ws.onopen = () => {
+      if (disposed) { ws.close(); return; }
+      terminalWs = ws;
+      // Flush pending message if session history already arrived (ready)
+      // or if we're still resuming (history handler will set ready but WS is now open)
+      if ((sendStatus === 'ready' || sendStatus === 'resuming') && pendingMessage) {
+        if (sendStatus === 'resuming') sendStatus = 'ready';
```

ESLint

[error] Expected { after 'if' condition.

(curly)